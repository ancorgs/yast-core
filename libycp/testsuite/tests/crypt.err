Parsed:
----------------------------------------------------------------------
size (crypt ("John"))
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
size (cryptmd5 ("Paul"))
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
size (cryptbigcrypt ("George"))
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
size (cryptblowfish ("Ringo"))
----------------------------------------------------------------------
