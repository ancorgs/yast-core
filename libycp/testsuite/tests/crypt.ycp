
//
// various password encryption functions
//

( size (crypt ("John")) )

( size (cryptmd5 ("Paul")) )

( size (cryptbigcrypt ("George")) )

( size (cryptblowfish ("Ringo")) )

