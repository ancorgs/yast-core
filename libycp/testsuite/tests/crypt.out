(13)
(34)
(13)
(60)
